src/main/java/com/echainika/app/model/Error.java

```diff
@@ -0,0 +1,12 @@
+package com.echainika.app.model;
+
+
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class Error {
+    private Integer rowNumber;

```

Comment:
should be candidate id

Reply:
cannot be renamed as it signifies the row number in the excel sheet which was uploaded to update candidate information in the db